tests/integration/targets/vmware_vm_storage_policy/aliases

```diff
@@ -0,0 +1,2 @@
+needs/target/prepare_vmware_tests
+zuul/vmware/vcenter_only
```

Comment:
Please add:
`cloud/vcenter`

This will force `ansible-test` to use the `vcenter` driver. It will set the correct value for `hostname`, `username` and `password`.

Reply:
Fixed in https://github.com/ansible-collections/vmware/pull/237/commits/c73f16830ade60996cc89af9b95ce818b2775e21 .  I'm also assuming it was related to your request for "recheck" @goneri ?  Thanks!

Reply:
Yes, hopefully, this is the last change before we get a green CI! :crossed_fingers: